docs/plugins/nist_nvd/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: blackstork/nist_nvd
+weight: 20
+plugin:
+  name: blackstork/nist_nvd
+  description: ""
+  tags: []
+  version: "v0.4.0"
+  source_github: "https://github.com/blackstork-io/fabric/tree/main/internal/nist_nvd/"
```

Comment:
wait, it is `internal/nist_nvd` here

Reply:
I think it's wrong here to expect source code directory name to match exactly plugin names. Maybe we should update templates and provide source dir separately or retrieve it using reflection.

Reply:
agreed, the source code URL can be part of the plugin's metadata // @Andrew-Morozko

Reply:
@traut what about if we add simple `if` check for now in template until we have completed revamp on plugin specs?